uppsrc/CtrlCore/X11Ctrl.h

```diff
@@ -94,7 +94,9 @@
 
 	virtual void    EventProc(XWindow& w, XEvent *event);
 	virtual bool    HookProc(XEvent *event);
-	Window  GetWindow() const         { return top ? top->window : None; }
+	Window  GetWindow() const         { return top && utop ? utop->window : None; }  //aris002
```

Comment:
IMO, no need for checking top and utop:
`Window GetWindow() const { return top ? utop->window : None; }`
should be sufficient. Please look at GetTop() and SetTop() implementation.